app/models/experimentation/experiment.rb

```diff
@@ -0,0 +1,21 @@
+module Experimentation
+  class Experiment < ActiveRecord::Base
+    has_many :treatment_groups
+    has_many :patients, through: :treatment_groups
+
+    validates :name, presence: true, uniqueness: true
+    validates :state, presence: true
+    validates :experiment_type, presence: true
+
+    enum state: {
+      new: "new",
+      selecting: "selecting",
+      live: "live",
+      complete: "complete"
+    }, _prefix: true
+    enum experiment_type: {
+      current_patient_reminder: "current_patient_reminder",
+      stale_patient_reminder: "stale_patient_reminder"
```

Comment:
Name feedback: I think `current_patient` and `inactive_patient` may be more clear.  Or perhaps `stale_patient` if you want to avoid "inactive".

Reply:
I'm open to change this. Daniel has also been calling them "dusty" patients, but inactive is probably the most immediately understandable word choice.

Reply:
we can handle this in a follow up PR